histomicstk/FeatureExtraction.py

```diff
@@ -115,6 +112,24 @@ def FeatureExtraction(Label, I, K=128, Fs=6, Delta=8):
         MinorAxisLength = np.append(MinorAxisLength, region.minor_axis_length)
         Extent = np.append(Extent, region.extent)
         Solidity = np.append(Solidity, region.solidity)
+        # get bounds of dilated nucleus
+        # region.bbox : min_row, min_col, max_row, max_col
+        # bounds : min_row, max_row, min_col, max_col
+        bounds = GetBounds(region.bbox, Delta, size_x)
+        # grab nucleus mask
+        Nucleus = (
+            Label[bounds[0]:bounds[1], bounds[2]:bounds[3]] == region.label
+        ).astype(np.int)
+        # get Bounds
+        Bounds = np.argwhere(
+            find_boundaries(Nucleus, mode="inner").astype(np.uint8) == 1
+        )
+        # calculate FSDs
+        FSD = FSDs(
+            Bounds[:, 0], Bounds[:, 1],
+            K, Interval
+        )
+        FSDGroup = np.append(FSDGroup, FSD)
 
     # add columns to dataframe
     df['X'] = CentroidX
```

Comment:
CentroidX is a list of x points that indicate a center of each object, so it could have two values if there're two objects in an image. For example, if there is an image labeled with '1' and '2' for two different objects, CentroidX of the first object labeled with '1'  will be "51.3231" (for instance), while another CentroidX of the second object labeled with '2' will be "56.1515". As far as I know, one object will have only one value of CentroidX.